app/views/user_mailer/batch_subscription_notification.html.erb

```diff
@@ -6,11 +6,11 @@
     <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
     <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
 
-    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> posted a
+    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> <%= t 'user_mailer.batch_subscription_notification.posted_a' %> 
     <% if creation.is_a?(Work) %>
-      <%= this_work.backdate ? "backdated" : "new" %> work:
+      <%= if this_work.backdate then t 'user_mailer.batch_subscription_notification.backdated' else t 'user_mailer.batch_subscription_notification.new' end  %> <%= t 'user_mailer.batch_subscription_notification.work' %>
```

Comment:
This construct will result in a string that goes adjective-noun -- which will be a bummer for languages where adjectives follow the nouns, or even more fun, for where that will vary depending on what the adjective is (for example, in French the likely translation for "new work" would be constructed "new work" but "backdated work" would go "work backdated")

So these need to be separate strings for "backdated work" and "new work".